src/zenml/models/filter_models.py

```diff
@@ -278,12 +278,26 @@ class BaseFilterModel(BaseModel):
 
     @validator("sort_by", pre=True)
     def sort_column(cls, v: str) -> str:
-        """Validate that the sort_column is a valid filter field."""
-        if v in cls.FILTER_EXCLUDE_FIELDS:
+        """Validate that the sort_column is a valid column with a valid operand."""
+        column = v
+        split_value = v.split(":", 1)
+        if (
+                len(split_value) == 2
+        ):
+            column = split_value[1]
+
+            if split_value[0] not in SorterOps.values():
+                logger.warning("Invalid operand used for column sorting."
+                               "Only 'asc' and 'desc' are supported."
+                               "Defaulting to 'asc' on column `%s`.",
```

Comment:
Maybe use `SorterOps.values()` in the warning message as well to cover future extensions?